emr/emr_config.yml

```diff
@@ -3,10 +3,13 @@ master_instance:
 core_instances:
   count: 2
   type: r5.2xlarge
-s3_scratch: s3://nrel-dsgrid-int-scratch/test-emr
+
+# profile: nrel-aws-dsgrid
 ssh_keys:
-  pkey_location: /Users/roliveir/.ssh/dsgrid-key.pem # The path to your private key file used with AWS.
-  key_name: dsgrid-key # The name of the permission key file on AWS.
+  pkey_location: /Users/lliu2/.ssh/lliu2-dsgrid.pem # The path to your private key file used with AWS.
```

Comment:
`~/.ssh/dsgrid.pem` should work with your code in the script. It's better to not force users to change. They can adopt this naming convention.